topsy.el

```diff
@@ -75,6 +75,16 @@ nil key defines the default function."
   :type '(alist :key-type symbol
                 :value-type function))
 
+(defcustom topsy-previous-line-fallback t
+  "Show line above the window start instead of blank header."
+  :type '(choice (const :tag "Show previous line" t)
+                 (const :tag "Leave header blank" nil)))
+
+(defface topsy-highlight '((t :weight bold :underline t))
```

Comment:
Do we need to enable the underline by default?  Seems like that could look a bit strange.